daprdocs/content/en/reference/api/scheduler_api.md

```diff
@@ -0,0 +1,109 @@
+---
+type: docs
+title: "Scheduler API reference"
+linkTitle: "Scheduler API"
+description: "Detailed documentation on the scheduler API"
+weight: 1300
+---
+
+{{% alert title="Note" color="primary" %}}
+Scheduler is currently in alpha.
+{{% /alert %}}
+
+With the Scheduler API, you can orchestrate jobs and tasks in your environment.
+
+## Schedule a job
+
+Schedule a job with the given name.
+
+```
+POST http://localhost:3500/v1.0-alpha1/job/schedule/<name>
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`name` | Job name
+
+### Request content
+
+Any request content will be passed to the scheduler as input. The Dapr API passes the content as-is without attempting to interpret it.
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Request formatted correctly, error in dapr code or Scheduler control plane service
+
+### Response content
+
+The API call will provide a response similar to this:
+
+```json
+```
+
+## Get job data
+
+Get a job's data with its given name.
+
+```
+GET http://localhost:3500/v1.0-alpha1/job/<name>
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`name` | Job name
+
+### HTTP response codes
+
+Code | Description
+---- | -----------
+`202`  | Accepted
+`400`  | Request was malformed
+`500`  | Request formatted correctly, error in dapr code or Scheduler control plane service
+
+### Response content
+
+The API call will provide a response similar to this:
+
+```json
```

Comment:
Actually, just saw this. Rn, if I schedule/get/delete a job successfully this is what I get:
```
$ curl -X POST \
  http://localhost:3500/v1.0-alpha1/job/schedule/test1 \
  -H "Content-Type: application/json" \
    -H 'dapr-app-id: sub' \
  -d '{
        "job": {
            "data": {
                "@type": "type.googleapis.com/google.type.Expr",
                "expression": "cassie87"
            },
            "dueTime": "30s"
        }
    }'

$ curl -X GET http://localhost:3500/v1.0-alpha1/job/test1 -H "Content-Type: application/json" -H 'dapr-app-id: sub'
{"name":"test1","dueTime":"30s","data":{"@type":"type.googleapis.com/google.type.Expr","expression":"cassie87"}}%                                    

$ curl -X DELETE http://localhost:3500/v1.0-alpha1/job/test1 -H "Content-Type: application/json" -H 'dapr-app-id: sub'
```

Because the return types are `google.protobuf.Empty` for the schedule & delete